docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>Can I ask for one or two sentences explaining what you're computing here and what's the information in the table? I can follow it because I'm familiar with what's going on, but I think it would be better if it was more digested for users.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>